packages/orchestration/src/joiner/helpers.ts

```diff
@@ -0,0 +1,59 @@
+import { RemoteJoinerQuery } from "@medusajs/types"
+
+export function toRemoteJoinerQuery(obj: any): RemoteJoinerQuery {
+  const remoteJoinerQuery: RemoteJoinerQuery = {
+    alias: "",
+    fields: [],
+    expands: [],
+  }
+
+  let isEntryPoint = true
```

Comment:
suggestion: can make it the third args of extractRecursive with true as a default value